cpp/arcticdb/storage/lmdb/lmdb_storage.hpp

```diff
@@ -16,7 +16,12 @@
 #include <folly/Range.h>
 #include <arcticdb/util/composite.hpp>
 
-#include <third_party/lmdbxx/lmdb++.h>
+#ifdef ARCTICDB_USING_CONDA
+    #include <lmdb++.h>
+#else
+    #include <third_party/lmdbxx/lmdb++.h>
```

Comment:
Yes, it would be better having this handled by CMake.

I will have a look at it on Friday.

Reply:
In order to have a proper resolution, we need to add a `CMakeLists.txt` to `third/lmdbxx` containing:

```cmake
target_include_directories(
  lmdbxx PUBLIC
  $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
  $<INSTALL_INTERFACE:include>  # <prefix>/include
)
```

but since [`third/lmdbxx` is a git submodule](https://github.com/man-group/ArcticDB/blob/master/.gitmodules#L7-L9), we will either:
 - need to maintain a fork for this revision
 - do a PR to the `lmdbxx` to bring a CMake setup
 
Both options seems to be heavier than having this ugly yet functional pre-processing in the meantime.

What do you think, @qc00 and @mehertz?

Reply:
Presumably we could also `include_directories` and manually add the files as source files. That's also a bit rubbish.

Okay, feels like we don't have much choice. I'm okay with this.